app/views/api/v1/messages/_message.json.jbuilder

```diff
@@ -0,0 +1,7 @@
+json.message_id message.id
+json.content message.content
+json.user do
+  json.user_id message.user.id
+  json.username message.user.username
+end
+json.created_at message.created_at
```

Comment:
Could you try updating this like the apiary example? WDYT?
```suggestion
json.id message.id
json.content message.content
json.user do
  json.id message.user.id
  json.username message.user.username
end
json.date message.created_at
```

Reply:
Done!